src/zenml/integrations/whylogs/materializers/__init__.py

```diff
@@ -0,0 +1,16 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+from zenml.integrations.whylogs.materializers.whylogs_materializer import (  # noqa
+    WhylogsMaterializer,
```

Comment:
Is this a necessary import? The base __init__ already does it

Reply:
I believe this is useful for people who want to import this materializer directly from the integration.

Reply:
Also, the base init should only import it when the integration gets activated.

Reply:
If by the base init, you mean `zenml.integrations.whylogs`, then no, I didn't add it there. But you're right, it's not really needed.

Reply:
Let's then leave it as it is, ill resolve all related comments